README.md

```diff
@@ -50,3 +58,14 @@ It also shows how you can modify your pre-built Stratux US version to run the EU
 
 If you want to customize beyond that, or have different needs, you can find a full list of supported hardware/attachments [here](https://github.com/stratux/stratux/wiki/Supported-Hardware).
 
+## In-system upgrade process
```

Comment:
Who might be interested in reading such a description? Probably only someone who is actively working on something related to the update process.
-> Rather put this either in a comment e.g. in the pre-start script, or in the Wiki or something. But not the main readme.
If we put all these implementation details in there, it would probably be 10k lines and nobody would read it anyway.

Reply:
I can put it under a developers heading but I’m convinced we should put developer info in the readme, for version control, review, and visibility.

Reply:
We did a poll on that and it disagreed. If you really really really want to have that in the main git repository, put it in a readme in some sub directory, or as I suggested as a comment in the code/script (imo the best place).
The main readme is really not the place for such detailed inner working description of an internal process.

Reply:
I saw the poll but as those people aren’t developers they don’t really understand the question or consequences.

I can out it in a developers.md file.